Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -200,6 +200,9 @@ instance {L : Type*} [Field k] [Field L] [Algebra k L] [Algebra.IsAlgebraic k L]
   isAlgebraic := .trans k L _
   isAlgClosed := inferInstance
 
+theorem aroots_card_eq_natDegree {p : k[X]} : (p.aroots <| AlgebraicClosure k).card = p.natDegree :=
```

Comment:
Do we really want this lemma? Is `IsAlgClosed.aroots_card_eq_natDegree` not enough?

Reply:
This version doesn't require `Field` and `IsAlgClosed` and `FaithfulSMul` because those are known about `AlgebraicClosure`, but I deleted it